spec/decorator_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './trimmer_decorator'
+require './capitalize_decorator'
+require './person'
+
+describe Decorator do
+  before :each do
+    @person = Person.new 23, 'antetokounmpo'
+    @capitalize = CapitalizeDecorator.new(@person)
+    @trimmer_decorator = TrimmerDecorator.new(@capitalize)
+  end
+
+  it 'Should display correct name' do
+    expect(@person.correct_name).to eq 'antetokounmpo'
+  end
+
+  it 'should CAPS person name' do
+    expect(@capitalize.correct_name).to eq 'ANTETOKOUNMPO'
+  end
+
+  it 'should slice if string is more than 10 char' do
+    expect(@trimmer_decorator.correct_name).to eq 'ANTETOKOUN'
+    expect(@trimmer_decorator.correct_name.length).to be <= 10
+  end
+end
```

Comment:
- [ ] Great job so far, since we are aiming for 100% test coverage, and it is also a [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md) for this project, " `Add unit tests to your classes: all classes that you created while implementing the Decorator design pattern` ", please let's cover the other classes that were created while implementing the Decorator design pattern with tests, classes like: [trimmer_decorator.rb](https://github.com/rolandfbk/oop-school-library/blob/unit-test/trimmer_decorator.rb), [capitalize_decorator.rb](https://github.com/rolandfbk/oop-school-library/blob/unit-test/capitalize_decorator.rb), [nameable.rb](https://github.com/rolandfbk/oop-school-library/blob/unit-test/nameable.rb). Please kindly create new test files for each of the above mentioned classes as you have done for the other ones.